stock_average_daily_sale/__manifest__.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock Average Daily Sale",
+    "summary": """
+        Allows to gather delivered products average on daily basis""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
```

Comment:
For the work on the conception
```suggestion
    "author": "ACSONE SA/NV,BCIM,Odoo Community Association (OCA)",
```